### ISSUE TITLE
Raise error/warning when a wrong setting is given in the cmdline args

### ISSUE THREAD

**Issue by yash-nisar** [coala/coala#4446]
For example, while running the given command, 
```
coala -b SpaceConsistencyBear -S random_value=True -f xyz.py --flush-cache
```
coala doesn't raise an error/warning for the wrong setting (`random_value=True`).

**Comment by Makman2** [comment]
a setting can't be "wrong", just unused. But we may warn the user that this might be the case.

**Comment by yash-nisar** [comment]
Yes exactly, something like `random_value` setting could not be found just like we do for bears ?

**Comment by yash-nisar** [comment]
Something similar to : 
```
No bears matching 'xyzBear' were found. Make sure you have coala-bears installed or you have typed the name correctly.
```

**Comment by Makman2** [comment]
"setting XYZ could not be found" sounds somehow critical. I would really suggest to use a warning with
"Setting XXX is not used by any bear, so it's meaningless." or similar

**Comment by CCInCharge** [comment]
Started looking through this - I'll take this on if no one else is working on it

**Comment by newbazz** [comment]
I would like to work on this if no one is presently working.

**Comment by Makman2** [comment]
Looks like there was recent activity ;)

**Comment by CCInCharge** [comment]
Pretty much there, just need to review it
